packages/lesswrong/components/common/Home2.jsx

```diff
@@ -28,18 +29,7 @@ class Home2 extends PureComponent {
         <HomeLatestPosts />
 
         <SingleColumnSection>
-          <SectionTitle title="Recent Discussion">
-            {shortformFeedId &&  <div onClick={this.toggleShortformFeed}>
-              <SectionButton>
-                <AddBoxIcon />
-                New Shortform Post
-              </SectionButton>
-            </div>}
-          </SectionTitle>
-          {showShortformFeed && <CommentsNewForm
-              post={{_id:shortformFeedId}}
-              type="comment"
-            />}
+          <SectionTitle title="Recent Discussion" />
```

Comment:
Is this EA Forum specific?

Reply:
No, except in as much as I think that was a merge mistake.